i18n/po/i18n_nl.po

```diff
@@ -22,6 +22,21 @@ msgstr ""
 msgid "Assertion violation processing RM schemas; original recipient: {0}"
 msgstr ""
 
+#: ../tools/src/main/java/com/nedap/archie/rmobjectvalidator/RMObjectValidationMessageIds.java:13
+#, java-format
+msgid "Attribute does not match cardinality {0}"
+msgstr "Attribuut moet {0} keer voorkomen"
```

Comment:
This should be something like:
'attribuut moet X waarden bevatten'
It shouldn't occur this many times, but the attribute should contain this many values...

For example, if there are two ways of recording some information, and only one choice be used, making the cardinality {1} but adding two CComplexObjects means a choice between the two CComplexObjects